### PR TITLE
dismiss-review

### PR DESCRIPTION
**Pull request by gk-sk**

### CODE REVIEW

File: web-pages/new-feature.html [hunk 1]
```diff
@@ -120,6 +120,7 @@
 	<div class="container">
 
 	   <h1>WATCH THIS VIDEO. coz, this is is really funny and hilarious but longer.</h1>
+     <p>Just another sentnece to dismiss-review.</p>
```
Comment (Gk1sK) [inline web-pages/new-feature.html:123]: this is added before.

**Review by Gk1sK — CHANGES_REQUESTED** [review]
this has been done before. So not sure if needed now.

**Review by Gk1sK — APPROVED** [review]
this has been done before. So not sure if needed now.

**Comment by gk-sk** [comment]
> this has been done before. So not sure if needed now.

thanks for the approval and additional request.

**Pull request merged**